app/Enum/EventRejectionReason.php

```diff
@@ -10,6 +10,6 @@ enum EventRejectionReason: string
     case NOT_APPLICABLE = 'not-applicable';
 
     public function getReason(): string {
-        return sprintf('notifications.eventSuggestionProcessed.%s', $this->value);
+        return __(sprintf('notifications.eventSuggestionProcessed.%s', $this->value));
     }
```

Comment:
Maybe remove this function? I don't see any reason to have this, as of the string is only used in one function... therefore we can reduce complexivity.

Reply:
imho it's cleaner to leave it herre. This way, we could change the values of the return types and still return the correct translation instead of guessing in several positions.